flake8_import_order/flake8_linter.py

```diff
@@ -12,8 +12,9 @@ class Linter(ImportOrderChecker):
     name = "import-order"
     version = __version__
 
-    def __init__(self, tree, filename):
+    def __init__(self, tree, filename, lines=None):
         super(Linter, self).__init__(filename, tree)
+        self.lines = lines
```

Comment:
This still doesn't work for flake8 2 when reading from stdin. stdin has already been read by this point, so the call to `load_file()` in `ImportOrderChecker`

```python
    def check_order(self):
        if not self.tree or not self.lines:
            self.load_file()
```

reads zero lines. `self.lines` is then empty list and so the `IndexError` reported in #81 still occurs at

```python
            if not pycodestyle.noqa(self.lines[import_.lineno - 1]):
                imports.append(import_)
```